examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.            ppc = pm.sample_posterior_predictive(trace=trace)
</pre><p> </p><p>I would directly add this to the inferencedata. I think the following should work.</p><p> </p><pre class="ql-syntax" spellcheck="false">trace.extend(az.from_pymc3(posterior_predictive=pm.sample_posterior_predictive(trace)))
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I did this and introduced `coords`. Everything seems to work but I get an odd message after the ppc sampling, namely the output of `In [6]` of the last commit says:
```
0, dim: steps, 300 =? 300
1, dim: y_, 3 =? 3
```

Reply:
This is a forgotten debug print that made it into latest arviz release, it has already been fixed in main but the new release is still pending